src/libraries/Directory.Build.targets

```
@@ -12,6 +12,7 @@
   <Import Project="..\..\Directory.Build.targets" />
 
   <PropertyGroup>
+    <TargetsForTfmSpecificContentInPackage Condition="'$(IsPackable)' == 'true'">LibIntellisenseDocs;$(TargetsForTfmSpecificContentInPackage)</TargetsForTfmSpecificContentInPackage>
```

nit: I would rather append than prepend to the list but you decide :)

```suggestion
    <TargetsForTfmSpecificContentInPackage Condition="'$(IsPackable)' == 'true'">$(TargetsForTfmSpecificContentInPackage);LibIntellisenseDocs</TargetsForTfmSpecificContentInPackage>
```